### PR TITLE
Development

### PR DESCRIPTION
**Pull request by htuerker**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,846 @@
...
+.news > .footnote:hover {
+    color: orangered;
+}
+
+
+.news-img {
+    position: relative;
+}
+
+.news-img {
```
Comment (heyapricot) [inline css/styles.css:278]: Don't duplicate selectors

File: index.html [hunk 2]
```diff
@@ -0,0 +1,882 @@
...
+            <!-- Brand and toggle get grouped for better mobile display -->
+            <div class="navbar-header w-100">
+                <div class="navbar-header w-100">
+                    <div class="date">Wed, Jan 23, 2019</div>
+                    <div class="logo w-100">Newsweek</div>
+                    <div class="right-links">
+                        <ul>
+                            <li><a href="#">SIGN IN</a></li>
+                            <li><a href="#">SUBSCRIBE ></a></li>
+                            <li> <button type="button" class="navbar-toggle pull-right" data-toggle="collapse"
```
Comment (heyapricot) [inline index.html:31]: What is this supposed to do? Nothing happens on my browser

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,846 @@
...
+hr {
+    margin: 10px 0;
+}
+
+.centered {
+    padding: 0 5px;
+    margin: 0 5px;
+}
+
+.w-100 {
```
Comment (heyapricot) [inline css/styles.css:15]: Doesn't bootstrap include a class for this?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,882 @@
...
+                    <p>The president tweeted that the students "were treated unfairly with early judgements proving out
+                        to be false."</p>
+                    <hr>
+                    <a href="" class="footnote">Twitter Suspends Account That Sparked Outrage</a>
+                    <hr>
+                </div>
+            </div>
+
+            <div class="col-xs-12 col-sm-7 col-md-6 col-lg-6">
+                <div class="row centered">
```
Comment (heyapricot) [inline index.html]: [`.flex-column`](https://getbootstrap.com/docs/4.1/utilities/flex/#direction) could make your life a little easier in the future

**Review by heyapricot — APPROVED** [review]
# You completed this project. Well done, looks very good 🙌🏼
That said, my main concern is some bad alignment you had for some spans on the main section 
screen shot 2019-01-28 at 10 51 09
Please address the comments and after that, you are ready to move on. No need from my side for further review.	
Tips:
For every HTML/CSS projects please check your files with the [W3 Validator](https://validator.w3.org) and the [CSS Linter](http://csslint.net/)

File: index.html (hunk 2 above)
Comment (htuerker) [reply index.html:31]: screen shot 2019-01-29 at 1 52 34 pm

File: css/styles.css (hunk 3 above)
Comment (htuerker) [reply css/styles.css:15]: It has container-fluid but it didn't work for us

**Pull request merged**